exist-core/src/test/java/org/exist/util/serializer/HTML5WriterTest.java

```diff
@@ -42,7 +42,7 @@ public void setUp() throws Exception {
 
     @Test
     public void testAttributeWithBooleanValue() throws Exception {
-        final String expected = "<!DOCTYPE html>\n<input checked>";
+        final String expected = "<!DOCTYPE html><input checked>";
```

Comment:
I am a bit confused. I thought the purpose of your change was to ensure that there was always a line break after the doctype declaration. Your changes to this test seem to show the opposite though.

Reply:
Yes, this is intentional. With `indent=no` the ultimate goal is to get minimal output. This is also how Saxon treats the option.